feature-libs/my-account/organization/core/services/organization-page-meta.resolver.ts

```diff
@@ -22,35 +30,79 @@ import { map } from 'rxjs/operators';
+
+  /**
+   * Translation key for the breadcrumb of Organization home page
+   */
+  protected readonly ORGANIZATION_TRANSLATION_KEY = 'organization.breadcrumb';
+
+  /**
+   * Semantic route name of the Organization home page
+   */
+  protected readonly ORGANIZATION_SEMANTIC_ROUTE = 'organization';
 
   constructor(
-    protected cms: CmsService,
+    protected contentPageMetaResolver: ContentPageMetaResolver,
     protected translation: TranslationService,
-    protected semanticPath: SemanticPathService
+    protected semanticPath: SemanticPathService,
+    protected routingService: RoutingService
   ) {
-    super(cms, translation);
+    super();
   }
+
   /**
-   * @override
-   * @returns {Observable<BreadcrumbMeta[]>} containing the localized label as well as the link for both home and organization breadcrumbs
+   * Breadcrumb of the Organization page.
+   * It's empty when the current page is the Organization page.
+   */
+  protected organizationPageBreadcrumb$: Observable<
+    BreadcrumbMeta[]
+  > = defer(() => this.routingService.getRouterState()).pipe(
+    map((routerState) => routerState?.state?.semanticRoute),
+    distinctUntilChanged(),
+    switchMap((semanticRoute) =>
+      semanticRoute === this.ORGANIZATION_SEMANTIC_ROUTE
+        ? of([])
+        : this.translation.translate(this.ORGANIZATION_TRANSLATION_KEY).pipe(
+            map((label) => [
+              {
+                label,
+                link: this.semanticPath.get(this.ORGANIZATION_SEMANTIC_ROUTE),
+              },
+            ])
+          )
+    )
+  );
+
+  /**
+   * Breadcrumbs returned in the method #resolveBreadcrumbs.
+   */
+  private breadcrumbs$: Observable<BreadcrumbMeta[]> = combineLatest([
+    this.organizationPageBreadcrumb$,
+    defer(() => this.contentPageMetaResolver.resolveBreadcrumbs()),
+  ]).pipe(
+    map(([organizationPageBreadcrumb, breadcrumbs]) => {
+      const [home, ...restBreadcrumbs] = breadcrumbs;
+      return [home, ...organizationPageBreadcrumb, ...restBreadcrumbs];
+    }),
+    shareReplay({ bufferSize: 1, refCount: true })
+  );
+
+  /**
+   * Returns list of breadcrumbs for:
+   * - the home page
+   * - the organization home page
+   * - the organization's child pages (i.e. cost center list)
+   * - sub-routes of the organization's child pages (i.e. cost center details, edit cost center, ...)
    */
   resolveBreadcrumbs(): Observable<BreadcrumbMeta[]> {
-    return combineLatest([
-      super.resolveBreadcrumbs(),
-      this.translation.translate(this.ORGANIZATION_TRANSLATION_KEY),
-    ]).pipe(
-      map(([breadcrumb, organizationLabel]: [BreadcrumbMeta[], string]) =>
-        breadcrumb.concat([
-          {
-            label: organizationLabel,
-            link: this.semanticPath.get('organization'),
-          },
-        ])
-      )
-    );
+    return this.breadcrumbs$;
+  }
+
+  resolveTitle(): Observable<string> {
+    return this.contentPageMetaResolver.resolveTitle();
```

Comment:
Good. i wouldn't mind to change this in the `ContentPageMetaResolver` and extend. Either way, i'm not opinionated.

Reply:
We will add more features to `ContentPageMetaResolver` and will benefit from this automatically here.